lib/mpl_toolkits/axes_grid1/parasite_axes.py

```diff
@@ -224,13 +199,9 @@ def get_aux_axes(self, tr, viewlim_mode="equal", axes_class=None):
         return ax2
 
     def _get_legend_handles(self, legend_handler_map=None):
-        # don't use this!
```

Comment:
@jklymak You put this coment in last year. Why? Should it be removed?

Reply:
https://github.com/matplotlib/matplotlib/pull/9324#discussion_r146154992  I *think* the whole method should be removed, but maybe I'll put that in as a separate PR and not burden @anntzer with sorting out this stuff as part of an otherwise straightforward PR